lib/ace/http/service.ex

```diff
@@ -118,9 +118,9 @@ defmodule Ace.HTTP.Service do
             Keyword.take(@socket_options ++ options, [:mode, :packet, :active, :reuseaddr])
 
           {:ok, listen_socket} = :gen_tcp.listen(port, tcp_options)
-          Logger.info("Serving cleartext using HTTP/1")
+          {:ok, port} = :inet.port(listen_socket)
+          Logger.info("Serving cleartext using HTTP/1 on port #{port}")
           {:tcp, listen_socket}
-
```

Comment:
This blank line was inserted by the new formatter so best not to delete it. please